### DISCUSSION TITLE
[FEATURE REQUEST]: SparkListener support

### DISCUSSION THREAD

**Discussion by bolcman** [dotnet/spark#590] [Q&A] [answered]
Any plan for support a handler like this ?

spark.sparkContext.addSparkListener(new SparkListener() {
  override def onTaskEnd(taskEnd: SparkListenerTaskEnd) {
    println("Task id: " + taskEnd.taskInfo.id.toString)
    println("Records written: " + taskEnd.taskMetrics.outputMetrics.recordsWritten.toString)
  }
})

**Comment by imback82** [comment, marked as answer]
@bolcman Thanks for the suggestion. Currently, there is no plan, but we can reprioritize based on the community demand.
